content/SCALE/SCALETutorials/Apps/InstallNextCloudMedia.md

```diff
@@ -102,6 +102,12 @@ Go to **Apps** to open the **Applications** screen and then click on the **Avail
 
    ![AddNextcloudEnableCronJobs](/images/SCALE/22.12/AddNextcloudEnableCronJobs.png "Nextcloud Enable CronJobs")
 
+{{< expand "Nextcloud Cron Jobs" "v" >}}
+NextCloud Cron Jobs only run while the app is running. If you stop the app, the Cron Job(s) will not run until you start the app again.
```

Comment:
present tense, active voice change suggestion
Also I don't think an expand is necessary here, just indent this information under this step number as supporting information for this step.
```suggestion
NextCloud Cron Jobs only run while the app is running. If you stop the app, the Cron Job(s) do not run until you start the app again.
```